lib/locales/fr.js

```diff
@@ -11,7 +11,64 @@
 module.exports = {
     moduleType: 'locale',
     name: 'fr',
-    dictionary: {},
+    dictionary: {
+        'Autoscale': 'Échelle automatique',
+        'Box Select': 'Sélection en boîte',
+        'Click to enter Colorscale titre': 'Cliquer ici pour ajouter un titre à l\'échelle de couleurs',
```

Comment:
@alexcjohnson and I are thinking about shortening this those _Cliquer ici pour ajouter_ to simply _Ajouter_, which should be clear enough. Any objections?

Reply:
done in https://github.com/plotly/plotly.js/pull/2252/commits/e3038cc3cb9b2796519ea836eeac212e5ef8be78